docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 45
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Mimir through the Prometheus data source type.
+
+## Before you begin
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. In the **HTTP** > **Prometheus server URL** field, enter a server URL:
+
+   - If you deployed Mimir via the `mimir-distributed` Helm chart,
+     the default URL is is `http://mimir-nginx.<MIMIR-NAMESPACE>.svc/prometheus`.
+
+   - If you deployed Mimir via Jsonnet and Tanka,
+     use `http://localhost:<MIMIR-PORT>`.
```

Comment:
I assume you need the `/prometheus` suffix here as well. But I don't use tanka, maybe ask @pstibrany

Reply:
I could move this to a follow-on issue to iterate more quickly. We can always add it back in, what do you think?

Reply:
```suggestion
     use `http://localhost:<MIMIR-PORT>/prometheus`.
```

Reply:
actually jsonnet/tanka is for kubernetes as well, so unless you do a port forward to localhost, this is not true

Reply:
We don't override `-http.prometheus-http-prefix` flag in Jsonnet, so it defaults to `/prometheus` there as well.